docs/home/contribute/create-pull-request.md

```diff
@@ -44,12 +44,30 @@ choose the
 [page type](/docs/home/contribute/page-templates/)
 that is the best fit for your content.
 
+## Spell Checking
+
+Check the spelling of any proposed changes. Verify any changes made by an automated spellchecker.
+
+You can check typos with the help of [Misspell](https://github.com/client9/misspell), which has been used in Kubernetes and Community project.
+
+1. If you use Go, you can install the Misspell tool by entering this command:
+    ```
+    go get -u github.com/client9/misspell/cmd/misspell
+    ```
+   Or you can install the tool [in other ways](https://github.com/client9/misspell#install).
+
+2. Then you can run this command under the root directory of your kubernetes/website repository:  
+    ```
+    git ls-files | grep -v generated |grep -v vendor | grep -v imported | grep -v api-reference | xargs misspell
+    ```
```

Comment:
I'm having trouble getting the expected results from this tool. I intentionally put several misspelled words in docs/tasks/run-application/delete-stateful-set.md.

"You can delet a StatefulSet in the samee way you delate oter ridiculus resorces in Kubernetes: use the `kubectl delete` command, and spesify the StatefulSet either by fiele or by name."

And yet the only thing that misspell detects is "ridiculus".

```
git ls-files | grep -v generated |grep -v vendor | grep -v imported | grep -v api-reference | xargs ~/bin/misspell
docs/tasks/run-application/delete-stateful-set.md:27:61: "ridiculus" is a misspelling of "ridiculous"
docs/user-journeys/users/cluster-operator/_advanced.md:13:278: "ridiculus" is a misspelling of "ridiculous"
```

Reply:
This tool use a word list to detect misspelling:
https://github.com/client9/misspell#where-do-the-word-lists-come-from

Reply:
`delet` and `samee` seems not in the dictionary. 
I set the command ignore the file which name contains `_advanced`. If you have any other file or folder that we should not check, plz let me know.